karma.py

```diff
@@ -1,4 +1,5 @@
 from brutal.core.plugin import BotPlugin, cmd, match
+from brutal.core.utils import change_cmd
```

Comment:
@pepol this whole thing always required `brutal-ng` to work =).

But yeah, the whole deployment should be described better somewhere, will do that at some point.